src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -393,6 +496,11 @@ public IA11yElement GetFirstChild()
             return Children[0];
         }
 
+        /// <summary>
+        /// Finds the first decendent element matching the given condition.
```

Comment:
Spelling: Should be "descendant" (has an 's' in the middle, ends in "ant" instead of "ent"

Usage Note from https://www.yourdictionary.com/descendent#americanheritage:

The adjective, "descending from a biological ancestor", may be spelt either with an a or with an e in the final syllable (see descendant). However the noun descendant, "one who is the progeny of someone", may be spelt only with an a.

```suggestion
        /// Finds the first descendant element matching the given condition.
```